account_sale_stock_report_non_billed/models/stock_move.py

```diff
@@ -53,6 +53,12 @@ def check_is_return(self):
     def get_total_devolution_moves(self):
         total_qty = 0
         for move in self:
+            # Avoid moves related to returns that not update qty on stock
+            if (
+                any(x.origin_returned_move_id for x in move.picking_id.move_lines)
```

Comment:
Why not `if move.origin_returned_move_id and not move.to_refund`? Each move can have a different selection.

Reply:
Yeah, you are right